.github/workflows/ci.yml

```diff
@@ -11,16 +11,17 @@ on:
 
 jobs:
   cabal:
-    name: ${{ matrix.os }} / ghc ${{ matrix.ghc }}
+    name: ${{ matrix.os }} / ghc ${{ matrix.ghc }} / cabal ${{ matrix.cabal }}
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
         os: [ubuntu-latest, macOS-latest, windows-latest]
-        cabal: ["3.6"]
+        cabal: ["3.6", "3.8"]
```

Comment:
We don't need to build with multiple `cabal` versions and display cabal in the job name. It's okay to use only the latest `cabal` version 👍🏻 

There's might be a reason not to do it when it doesn't work. But I expect things to be fine here.